ambry-messageformat/src/test/java/com.github.ambry.messageformat/BlobPropertiesTest.java

```diff
@@ -161,78 +138,55 @@ public void basicTest() throws IOException {
-   * @param serviceId the serviceId associated with the {@link BlobProperties}
-   * @param ownerId the ownerId associated with the {@link BlobProperties}
-   * @param contentType the contentType associated with the {@link BlobProperties}
-   * @param isPrivate refers to whether the blob is private or not
-   * @param timeToLiveInSeconds the time to live associated with the {@link BlobProperties} in secs
-   * @param accountId accountId of the user who uploaded the blob
-   * @param containerId containerId of the blob
-   * @param version the version in which {@link BlobProperties} needs to be created
-   * @return the {@link BlobProperties} thus created
-   */
-  private BlobProperties getBlobProperties(long blobSize, boolean isEncrypted, String serviceId, String ownerId,
-      String contentType, boolean isPrivate, long timeToLiveInSeconds, short accountId, short containerId,
-      short version) {
-    if (version == BlobPropertiesSerDe.VERSION_1) {
-      return new BlobProperties(blobSize, serviceId, ownerId, contentType, isPrivate, timeToLiveInSeconds,
-          Account.UNKNOWN_ACCOUNT_ID, Container.UNKNOWN_CONTAINER_ID, isEncrypted);
-    } else {
-      return new BlobProperties(blobSize, serviceId, ownerId, contentType, isPrivate, timeToLiveInSeconds, accountId,
-          containerId, isEncrypted);
-    }
-  }
-
-  /**
-   * Creates {@link BlobProperties} based on the args passed for the given version
-   * @param blobSize the size of the blob
-   * @param isEncrypted whether the blob is encrypted
-   * @param serviceId the serviceId associated with the {@link BlobProperties}
-   * @param accountId accountId of the user who uploaded the blob
-   * @param containerId containerId of the blob
-   * @param version the version in which {@link BlobProperties} needs to be created
-   * @return the {@link BlobProperties} thus created
+   * Serialize {@link BlobProperties} using {@link BlobPropertiesSerDe} in the given version
+   * @param blobProperties {@link BlobProperties} that needs to be serialized
+   * @param version the version to serialize
+   * @return the {@link ByteBuffer} containing the serialized {@link BlobPropertiesSerDe}
    */
-  private BlobProperties getBlobProperties(long blobSize, boolean isEncrypted, String serviceId, short accountId,
-      short containerId, short version) {
-    if (version == BlobPropertiesSerDe.VERSION_1) {
-      return new BlobProperties(blobSize, serviceId, Account.UNKNOWN_ACCOUNT_ID, Container.UNKNOWN_CONTAINER_ID);
-    } else if (version == BlobPropertiesSerDe.VERSION_2) {
-      return new BlobProperties(blobSize, serviceId, accountId, containerId);
-    } else {
-      return new BlobProperties(blobSize, serviceId, null, null, false, Utils.Infinite_Time,
-          SystemTime.getInstance().milliseconds(), accountId, containerId, isEncrypted);
+  private ByteBuffer serializeBlobPropertiesInVersion(BlobProperties blobProperties, short version) {
```

Comment:
minor: the version can be omitted and the instance variable can be used since you only deal with one version in a particular test.